### PR TITLE
Feat: show disabled strategies in playground

### PR DESCRIPTION
**Pull request by andreas-unleash**
Show disabled strategies in separate list
Do not show any results in segments/constraints when disabled

Closes # [1-1506](https://linear.app/unleash/issue/1-1506/display-disabled-strategies-in-playground-page)
Screenshot 2023-10-18 at 13 13 48

### CODE REVIEW

File: frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/playgroundResultStrategyLists.tsx [hunk 1]
```diff
@@ -44,7 +48,22 @@ export const PlaygroundResultStrategyLists = ({
...
+                    titlePrefix
+                        ? titlePrefix.concat(' strategies')
+                        : 'Strategies'
+                } (${strategies?.length})`}</Typography>
+                <ConditionallyRender
+                    condition={Boolean(infoText)}
+                    show={
+                        <Typography
+                            variant={'subtitle2'}
+                            sx={{ ml: 1, mb: 2, color: 'text.secondary' }}
```
Comment (kwasniew) [inline frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/playgroundResultStrategyLists.tsx:61]: probably deserves a styled component at this point

File: frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/ConstraintExecution/ConstraintExecution.tsx [hunk 2]
```diff
@@ -11,6 +15,7 @@ import { ConstraintOk } from './ConstraintOk/ConstraintOk';
 interface IConstraintExecutionProps {
     constraints?: PlaygroundConstraintSchema[];
     input?: PlaygroundRequestSchema;
+    showExecutionResult?: boolean;
```
Comment (kwasniew) [inline frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/ConstraintExecution/ConstraintExecution.tsx:18]: boolean props are usually a component design smell. Can we create 2 separate single responsibility components: ConstraintsWithResult and ConstraintWithoutResults?

Comment (kwasniew) [reply frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/ConstraintExecution/ConstraintExecution.tsx:18]: components are functions and functions in general should avoid boolean parameters: https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior

Comment (andreas-unleash) [reply frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/StrategyExecution/ConstraintExecution/ConstraintExecution.tsx:18]: makes sense

File: frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/FeatureStrategyItem.tsx [hunk 3]
```diff
@@ -4,6 +4,8 @@ import { PlaygroundStrategySchema, PlaygroundRequestSchema } from 'openapi';
 import { StrategyExecution } from './StrategyExecution/StrategyExecution';
 import { StrategyItemContainer } from 'component/common/StrategyItemContainer/StrategyItemContainer';
 import { objectId } from 'utils/objectId';
+import { ConditionallyRender } from '../../../../../../../common/ConditionallyRender/ConditionallyRender';
```
Comment (kwasniew) [inline frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/FeatureStrategyItem.tsx:7]: dots left

**Review by kwasniew — APPROVED** [review]

**Pull request merged**